scripts/l2deploy.ts

```diff
@@ -0,0 +1,168 @@
+import config from './config';
+import l2config from './l2config';
+import { getAllContractAddresses } from '@scripts/utils/loadContracts';
+import { ethers } from 'hardhat';
+import '@nomiclabs/hardhat-ethers';
+
+const {
+  /// fees
+  MINT_FEE_BASIS_POINTS,
+  TIMELOCK_DELAY
+} = config;
```

Comment:
We no longer need this entire deploy file right? Seeing as the functions have been separated into 3 different scripts

Reply:
we no longer need the l2systemDeploy file